apps/condo/domains/acquiring/schema/fields/Recipient.js

```diff
@@ -9,6 +9,7 @@ const RECIPIENT_FIELDS_DEFINITION = {
     offsettingAccount: 'String',
     tin: 'String!',
     iec: 'String',
+    // NOTE(YEgorLu): rename to routingNumber
```

Comment:
better to add TODO to make all recipients models and bankAccounts to use the same naming